applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -526,24 +526,103 @@ Use the following configuration options to customize the number of supported bri
+      You have to consider several factors to decide how many Bluetooth LE bridged devices to support.
+      Every Bluetooth LE bridged device uses a separate Bluetooth LE connection, so you need to consider the number of supported Bluetooth LE connections (selected using the :kconfig:option:`CONFIG_BT_MAX_CONN` Kconfig option) when you configure the number of devices.
+      Since the Matter stack uses one Bluetooth LE connection for commissioning, the maximum number of connections you can use for bridged devices is one less than is configured using the :kconfig:option:`CONFIG_BT_MAX_CONN` Kconfig option.
 
-   west build -b nrf7002dk/nrf5340/cpuapp -- -DCONFIG_BRIDGED_DEVICE_BT=y -DEXTRA_CONF_FILE="overlay-bt_max_connections_app.conf" -Dipc_radio_EXTRA_CONF_FILE="overlay-bt_max_connections_net.conf"
+      .. tabs::
 
-.. parsed-literal::
-   :class: highlight
+         .. group-tab:: Matter-Bridge over wifi configuration
+
+            Increasing the number of Bluetooth LE connections affects the RAM usage on both the application and network cores.
+            The current maximum number of Bluetooth LE connections that can be selected using the default configuration is ``10``.
+            You can increase the number of Bluetooth LE connections if you decrease the size of the Bluetooth LE TX/RX buffers used by the Bluetooth controller, but this will decrease the communication throughput.
+            Build the target using the following command in the project directory to enable a configuration that increases the number of Bluetooth LE connections to ``20`` by decreasing the sizes of Bluetooth LE TX/RX buffers:
+
+            .. parsed-literal::
+               :class: highlight
+
+               west build -b nrf5340dk/nrf5340/cpuapp -p -- -Dmatter_bridge_SHIELD=nrf7002ek -DSB_CONFIG_WIFI_PATCHES_EXT_FLASH_STORE=y -DSB_CONFIG_DFU_MULTI_IMAGE_PACKAGE_WIFI_FW_PATCH=y -DSB_CONFIG_WIFI_NRF700X=y -Dmcuboot_CONFIG_UPDATEABLE_IMAGE_NUMBER=3 -DCONFIG_BRIDGED_DEVICE_BT=y -DEXTRA_CONF_FILE="overlay-bt_max_connections_app.conf" -Dipc_radio_EXTRA_CONF_FILE="overlay-bt_max_connections_net.conf"
+
+
+         .. group-tab:: Matter-Bridge over thread configuration
+
+            Increasing the number of Bluetooth LE connections affects the RAM usage on both the application and network cores.
+            The current maximum number of Bluetooth LE connections that can be selected using the default configuration is ``10``.
+            Due to using thread and BLE protocols in this configuration You can't increase the number of Bluetooth LE connections and additionally you have to decrease the size of the Bluetooth LE TX/RX buffers used by the Bluetooth controller, this will decrease the communication throughput.
+            Build the target using the following command in the project directory to enable a configuration that allow to set the number of Bluetooth LE connections to ``10`` by decreasing the sizes of Bluetooth LE TX/RX buffers:
```

Comment:
Isn't it that we say here - the default configuration is 10, but you can configure it to use max number which is 10? :smile: